mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -250,28 +246,37 @@ def parse_webhook_data(self, event):
             # Acquiring labels specific to this repo
             labels = []
             actions = {}
+
+            # Looks for and reads phrase referencing @mxnet-label-bot
             if "@mxnet-label-bot" in payload["comment"]["body"]:
-                labels += self._tokenize(payload["comment"]["body"])
+                phrase = payload["comment"]["body"][payload["comment"]["body"].index("@mxnet-label-bot"):]
+
+                labels += self._tokenize(phrase)
                 if not labels:
-                    logging.error("Message typed by user: {}".format(payload["comment"]["body"]))
+                    logging.error(f'Message typed by user: {phrase}')
```

Comment:
same comment as above, should be ('Message typed by user: %s', phrase) to avoid uneccesary formatting

Reply:
I believe we should stick with the usage of the f string in this case as it is easier to read